docs/contributor_guide/index.rst

```diff
@@ -18,3 +18,69 @@ The contribution guide explains how to structure your contributions. Please
+- Be clear that AI systems (and technology in general) are never *neutral* –
+  all AI systems necessarily reflect the assumptions, priorities, and values
+  of the people involved in their development and deployment.
+- Be clear that prioritizing fairness in AI systems often means making
+  tradeoffs based on competing priorities, there are seldom clear-cut answers
+  (this is why using the word *solve* is seldom appropriate).
+- Be clear that prioritizing fairness in AI systems is a sociotechnical
+  challenge – it is not something that can be accomplished via purely
+  technical methods (or purely social methods, for that matter).
+- Be clear that there is no software tool that will *solve* fairness in all AI
+  systems – this is not to say that software tools don’t have a role to play,
+  but they will be precise, targeted, and only part of the picture.
+- Be clear that even with precise, targeted tools, it’s easy to overlook
+  things, especially things that are difficult to quantify – software tools
+  must be supplemented with other resources and processes.
+- There are many reasons why AI systems can behave unfairly, not just societal
+  biases. Also *bias* is ambiguous and means different things to different
+  communities – e.g., statistical bias vs. societal biases. For this reason,
+  talk about *fairness issues* or *fairness-related harms* rather than *bias*,
+  unless you are very specifically referring to societal biases (or
+  statistical bias or some other definition of bias).
+- Better yet, be specific about the type of fairness-related harm – is this a
+  harm of allocation, quality of service, stereotyping, denigration, or over-
+  or under-representation? (But be clear that different types of
+  fairness-related harm are not mutually exclusive; a single AI system can
+  exhibit more than one type)
+- Be clear that fairness-related harms can have varying severities, but that
+  the cumulative impact of even *non-severe* harms can be extremely burdensome
+  or make people feel singled out or undervalued.
+- Be clear that fairness-related harms can affect both the people who will use
+  an AI system and the people who will be directly or indirectly affected by
+  the system, either by choice or not. For this reason, when talking about the
+  people who might be harmed by a system, talk about *stakeholders* not
+  *users*.
+- When talking about who might be harmed, don’t just focus on demographic
+  groups (e.g., groups defined in terms of race, gender, age, disability
+  status, skin tone, and their intersections) or groups that are protected by
+  anti-discrimination laws – the most relevant groups may be context specific.
+- Be clear that stakeholders can belong to overlapping or intersectional
+  groups – e.g., different combinations of race, gender, and age – and
+  considering each group separately may obscure harms.
+- Talk about the *AI development and deployment lifecycle* rather than, e.g.,
```

Comment:
These sort of "linguistic choices" points might belong in their own bulleted section.

Reply:
This has all been rewritten now. Resolving the old comments because they refer to a completely different version. Let me know if you think this is better. I can always go back to the original.